exporter/kedascalerexporter/README.md

```diff
@@ -0,0 +1,71 @@
+# KEDA Scaler Exporter
+
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Stability     | beta: metrics   |
+| Distributions | contrib |
+| Code Owners   | [@sawmills](https://github.com/sawmills) |
```

Comment:
this will not work. We cannot assign organizations as code owners.

Reply:
I changed it to my user `ronkorland`